response/writer.go

```diff
@@ -53,12 +53,18 @@ func writeUnmodifiedResponse(ctx context.Context, w http.ResponseWriter, babbage
 }
 
 func writeResponseWithMaxAge(ctx context.Context, w http.ResponseWriter, babbageResponse *http.Response, maxAge int) {
-	cacheControlValue := fmt.Sprintf("max-age=%d", maxAge)
-	additionalHeaders := map[string]string{
-		"Cache-Control": cacheControlValue,
+	// Prepare the additional headers
```

Comment:
You can get rid of this comment, as it does not add any value (in fact it is wrong, "additional headers" is the old name).